packaging/rpm/microshift.spec

```diff
@@ -283,6 +283,10 @@ install -p -m644 assets/optional/operator-lifecycle-manager/0000* %{buildroot}/%
 install -p -m644 assets/optional/operator-lifecycle-manager/kustomization.yaml %{buildroot}/%{_prefix}/lib/microshift/manifests.d/001-microshift-olm
 install -p -m755 packaging/greenboot/microshift-running-check-olm.sh %{buildroot}%{_sysconfdir}/greenboot/check/required.d/50_microshift_running_check_olm.sh
 
+mkdir -p -m755 %{buildroot}%{_datadir}/microshift/release
+install -p -m644 assets/optional/operator-lifecycle-manager/release-aarch64.json %{buildroot}%{_datadir}/microshift/release/olm-aarch64.json
+install -p -m644 assets/optional/operator-lifecycle-manager/release-x86_64.json %{buildroot}%{_datadir}/microshift/release/olm-x86_64.json
```

Comment:
We need to add the following:
- Update the changelog, using 4.15.0 version so that we can easily backport
- Update `%files olm` to include these new files, or they are not deleted